vuepress/docs/next/tutorials/samples/entando-hub.md

```diff
@@ -130,7 +139,7 @@ A single Spring Boot microservice provides two REST endpoints:
 - The second provides methods that support the Entando App Builder integration (7.0.0+).
 
 ### Content
-The Hub content bundle includes a custom template and a page preconfigured with the main Hub micro frontends.
+The Hub content bundle (`entando-hub`) includes a custom template and a page preconfigured with the main Hub micro frontends.
```

Comment:
not a big deal, but is the main MFE a singular thing or plural?

Reply:
Plural, there are 2 currently.